mountain-climbing/src/main/java/com/example/mountainclimbing/model/Image.java

```diff
@@ -0,0 +1,26 @@
+package com.example.mountainclimbing.model;
+
+import java.time.LocalDateTime;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.relational.core.mapping.Table;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+
+@Table(value = "images")
+@Getter
+@Setter
+@RequiredArgsConstructor
+public class Image {
+
+	@Id
```

Comment:
Add validation annotations for each field. For exampe NotNull, Number, Regex, Size etc